h/static/scripts/directive/annotation.js

```diff
@@ -655,8 +655,8 @@ function AnnotationController(
   vm.share = function(event) {
     var $container = angular.element(event.currentTarget).parent();
     $container.addClass('open');
-    var shareLinkInput = $container.find('input')[0];
-    shareLinkInput.focus();
+    var shareLinkInput = $container.find('.annotation-share-dialog-link__text');
```

Comment:
The logic for selecting the input field should live in the share dialog, rather than in the parent annotation card.